src/forecast.js

```diff
@@ -0,0 +1,19 @@
+const key = 'Fa1tORczSKnflphuiPFCKU303Z04wWn9';
+
+const getWeather = async (locationid) => {
```

Comment:
## Add a new module for the `toggle` functionality
- [ ] This project is required to have toggle functionality in place. There is an excellent effort in displaying the temperature in both versions (celsius and Fahrenheit). 
![temp-toggle](https://user-images.githubusercontent.com/22345961/107818916-5b060380-6d89-11eb-8e73-c8af8c24bdc2.png)
In this instance, there were two options to go about this:

- Either make two different requests for the `Celsius data` and the `Fahrenheit data` OR make only one request and use math to convert it to the other unit as expressed here in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app). 
- It is great you chose to go with the former, now let's add the toggle functionality. As a user, I need to toggle in between the `temperature` values.